airbyte-integrations/connectors/destination-mongodb/src/main/resources/spec.json

```diff
@@ -0,0 +1,90 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/destinations/mongodb",
+  "supportsIncremental": true,
+  "supportsNormalization": false,
+  "supportsDBT": false,
+  "supported_destination_sync_modes": ["overwrite", "append"],
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "MongoDB Destination Spec",
+    "type": "object",
+    "required": ["host", "port", "database", "auth_type"],
+    "additionalProperties": true,
+    "properties": {
+      "host": {
+        "title": "Host",
+        "description": "Hostname of the database.",
+        "type": "string",
+        "order": 0
+      },
+      "port": {
+        "title": "Port",
+        "description": "Port of the database.",
+        "type": "integer",
+        "minimum": 0,
+        "maximum": 65536,
+        "default": 27017,
+        "examples": ["27017"],
+        "order": 1
+      },
+      "database": {
+        "title": "DB Name",
+        "description": "Name of the database.",
+        "type": "string",
+        "order": 2
+      },
+      "auth_type": {
+        "title": "Authorization type",
+        "type": "object",
+        "description": "Authorization type.",
+        "oneOf": [
+          {
+            "title": "None",
+            "additionalProperties": false,
+            "description": "None.",
+            "required": ["authorization"],
+            "type": "object",
+            "properties": {
+              "authorization": {
+                "type": "string",
+                "description": "Without authorization.",
+                "enum": ["none"],
```

Comment:
Please consider using "const" here instead of Enum https://docs.airbyte.io/connector-development/connector-specification-reference#using-oneofs

Reply:
Yes, it looks so much better